observability/src/test/java/io/grpc/observability/LoggingTestHelper.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2022 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.observability;
+
+import io.grpc.ServerCallHandler;
+import io.grpc.stub.StreamObserver;
+import io.grpc.testing.protobuf.SimpleRequest;
+import io.grpc.testing.protobuf.SimpleResponse;
+import io.grpc.testing.protobuf.SimpleServiceGrpc;
+import org.mockito.ArgumentMatchers;
+
+public class LoggingTestHelper {
+
+  static ServerCallHandler<String, Integer> anyCallHandler() {
+    return ArgumentMatchers.any();
+  }
+
+  static String unaryRpc(
```

Comment:
This static method called `unaryRpc` is a bit misleading. What it does is to make a unary call thru a client stub. How about naming it as `makeUnaryRpcViaClientStub` or something like that?

Reply:
Renamed `unaryRpc` to `makeUnaryRpcViaClientStub`